tests/integration/test.py

```diff
@@ -0,0 +1,67 @@
+from contextlib import contextmanager
```

Comment:
should this file be `tests/integration/test_max_rows_to_read_leaf_with_view/test.py`?

Reply:
> should this file be `tests/integration/test_max_rows_to_read_leaf_with_view/test.py`?

🤦  Yes, it should. Sorry about that @hanfei1991 . Updated my branch.